README.md

```diff
@@ -97,6 +97,17 @@ python verify_dynamo.py
 
 If you do not have the PyTorch source locally, you can alternatively copy the script (`tools/dynamo/verify_dynamo.py`) from the PyTorch repo and run it locally.
 
+## Docker installation
+
+We also provide all the required dependencies in the PyTorch nightly binaries which you can download with
+
+`docker pull ghcr.io/pytorch/pytorch-nightly`
+
+And for ad hoc experiments just make sure that your container has access to all your GPUs
+
+`docker run --rm --gpus all -it pytorch-nightly /bin/bash`
```

Comment:
I wonder if it's better to omit `--rm` option here, as it allows folks to continue work from an accidentally stopped container